packages/installations/src/functions/get-id.ts

```diff
@@ -28,14 +27,11 @@ export async function getId(app: FirebaseApp): Promise<string> {
   );
 
   if (registrationPromise) {
-    // Suppress registration errors as they are not a problem for getId.
-    registrationPromise.catch(() => {});
-  }
-
-  if (installationEntry.registrationStatus === RequestStatus.COMPLETED) {
+    registrationPromise.catch(console.error);
+  } else {
     // If the installation is already registered, update the authentication
-    // token if needed. Suppress errors as they are not relevant to getId.
-    refreshAuthToken(appConfig).catch(() => {});
+    // token if needed.
+    refreshAuthToken(appConfig).catch(console.error);
```

Comment:
Huh, this never used to work without a .bind(console)

Reply:
The future is now.